aspnetserver/Services/UtenteService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using aspnetserver.Models;
+using aspnetserver.Resources;
+using Microsoft.EntityFrameworkCore;
+
+namespace aspnetserver.Services
+{
+    public sealed class UtenteService : IUtenteService
+    {
+        private readonly CarrelloSpesaContext _context;
+        private readonly string _pepper;
+        private readonly int _iteration = 3;
+
+        public UtenteService(CarrelloSpesaContext context)
+        {
+            _context = context;
+            _pepper = Environment.GetEnvironmentVariable("PasswordHashExamplePepper");
+        }
+
+        public async Task<UtenteResource> Register(RegisterResource resource, CancellationToken cancellationToken)
```

Comment:
why CancellationToken ? are we passing a cancellation token or just a normal token ?